samila/genimage.py

```diff
@@ -184,27 +185,32 @@ def nft_storage(
         :type upload_config: bool
         :param depth: image depth
         :type depth: float
+        :param timeout: upload timeout (in seconds)
+        :type timeout: int
         :return: result as dict
         """
         save_params_filter(self, depth)
         response = save_fig_buf(self.fig, self.depth)
         if not response["status"]:
             return {"status": False, "message": response["message"]}
         buf = response["buffer"]
-        response = nft_storage_upload(api_key=api_key, data=buf.getvalue())
+        response = nft_storage_upload(
+            api_key=api_key,
+            data=buf.getvalue(),
+            timeout=timeout)
         if upload_config == False and upload_data == False:
             return response
         result = {key: {'image': value} for key, value in response.items()}
         if upload_config:
             response = nft_storage_upload(
                 api_key=api_key,
-                data=json.dumps(get_config(self)))
+                data=json.dumps(get_config(self)), timeout=timeout)
             for key, value in response.items():
                 result[key]['config'] = value
         if upload_data:
             response = nft_storage_upload(
                 api_key=api_key,
-                data=json.dumps(get_data(self)))
+                data=json.dumps(get_data(self)), timeout=timeout)
```

Comment:
Shift `timeout=timeout` to the next line.

Reply:
Is it longer that the pep8 limit? Take a look at the upper lines.

Reply:
No, it's not. But since the previous line has a parameter it seems right if we keep one parameter per line for this method.

Reply:
fix in https://github.com/sepandhaghighi/samila/pull/182/commits/a913d8c7e95b3777df719da3f9ddebcad14d1af9